meson.build

```diff
@@ -10,8 +10,8 @@ project('systemd', 'c',
         meson_version : '>= 0.40',
        )
 
-libsystemd_version = '0.19.0'
-libudev_version = '1.6.6'
+libsystemd_version = '0.19.1'
+libudev_version = '1.6.7'
```

Comment:
The last patch that changes exported symbols is v233-183-g19fe49f62c, so only bumping micro version is correct.